src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,37 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService service =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+
+        User chris = service.register("Chris_Pain@gmail.com", "1478963");
+        System.out.println(userService.findByEmail("Chris_Pain@gmail.com"));
+
+        User veronica = service.register("Veronica_Porter@gmail.com", "ohjrlio");
+        System.out.println(userService.findByEmail("Veronica_Porter@gmail.com"));
+
+        ShoppingCartService cartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+        cartService.addSession(tomorrowMovieSession, chris);
+        cartService.addSession(tomorrowMovieSession, veronica);
+        cartService.addSession(yesterdayMovieSession, chris);
+
+        ShoppingCart byUser = cartService.getByUser(chris);
+        byUser.getTickets().stream()
+                .map(Ticket::getMovie)
+                .forEach(System.out::println);
+
+        cartService.clearShoppingCart(cartService.getByUser(chris));
     }
```

Comment:
Maybe use some methods from orderService to test it?

Reply:
I'he alredy checked result in workbench.